spec/save_operation_spec.cr

```diff
@@ -75,6 +79,18 @@ describe "Avram::SaveOperation" do
     bucket.names.should eq([] of String)
   end
 
+  it "ignores empty params on nilable fields" do
```

Comment:
I'm not 100% sure I understand the "ignores" part. Does this mean that it treats empty params as nil without erroring? Just trying to make sure I get the behavior

Reply:
yeah, it's bad wording... Maybe `it "doesn't fail with empty string params on nilable fields"`?

Reply:
I like that I think adding what it does might clarify it even more. `" treats empty strings as `nil` for Time types instead of failing to parse"`?